diff -ur a/public_html/lists/admin/login.php b/public_html/lists/admin/login.php
--- a/public_html/lists/admin/login.php	2017-02-13 13:38:10.000000000 +0100
+++ b/public_html/lists/admin/login.php	2017-03-23 22:05:53.924636055 +0100
@@ -126,6 +126,6 @@
     echo '    <tr><td><input class="submit" type="submit" name="process" value="'.$GLOBALS['I18N']->get('Continue').'" /></td></tr>';
     echo '  </table>';
     echo '</form>';
-    footer();
+    if ($GLOBALS['disable_forgotpassword'] != 1) footer();
 }
 ?>
diff -ur a/public_html/lists/config/config_extended.php b/public_html/lists/config/config_extended.php
--- a/public_html/lists/config/config_extended.php	2017-03-23 22:12:49.799877671 +0100
+++ b/public_html/lists/config/config_extended.php	2017-03-23 22:12:19.187007015 +0100
@@ -140,6 +140,9 @@
 // set this to 1 if you still want to manage admin users despite using an external authentication
 $manage_admin = 1;
 
+// set this to 1 if you want to disable forgot password when using an external authentication
+$disable_forgotpassword = 1;
+
 // if you use login, how many lists can be created per administrator
 define('MAXLIST', 1);
 
